services/discourse_translator/google.rb

```diff
@@ -144,11 +144,19 @@ def self.result(url, body)
 
       if response.status != 200
         if body && body["error"]
-          raise TranslatorError.new(body["error"]["message"])
+          ProblemCheckTracker[:translator_error].problem!(
+            details: {
+              provider: "Google",
+              code: body["error"]["code"],
+              message: body["error"]["message"],
+            },
+          )
+          raise ProblemCheckedTranslationError.new(body["error"]["message"])
         else
           raise TranslatorError.new(response.inspect)
```

Comment:
Same here actually. We want to get rid of the log noise in our logs from misconfigured customer settings.

Reply:
For this `ProblemCheckedTranslationError`, we'll rescue it silently, so there won't be any logging issues. :wink: . yeah I think maybe it should have a better name, since it's hard for people to tell the first time they read it that this error won't be logged.

https://github.com/discourse/discourse-translator/blob/add-problem-check-google/plugin.rb#L131-L133

Reply:
Very good. 👍